docs/variants/msi_z690/recovery.md

```diff
@@ -44,3 +44,66 @@ $ flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7D25v13/E7D25IMS.13
 ```
 
 * First boot after the recovery process is significantly longer
+
+### SMBIOS unique data recovery
+
+#### Serial number
+
+SMBIOS specification defines two spaces for serial number: the system serial
+number and baseboard serial number. The original MSI PRO Z690-A firmware
+provides only baseboard serial number.
+
+In case you have lost your serial number in the process of flashing Dasharo or
+newer MSI firmware, there is a way to retrieve it. The board has a QR code
+imprinted between the chipset heatsink and dPGU PCIe slot:
+
+![](/images/msi_sn_qr.jpg)
+
+If you read the QR code with your smartphone you will get the full serial
+number. The serial number has the format `07D25xx_LyzEaaaaaa` where:
+
+* `07D25` - is the board model, i.e. MS-7D25 for this particular board
+* `xx` is the mainboard revision which should match the revision imprinted
+  between the M2_1 slot and dGPU slot. E.g. `xx=11` means VER:1.1
+* `yz` is the manufacturing date in hex, i.e. `y` is the month, `z` is the
+  year, for example `A1` means **October** 202**1**, `12` means **January**
+  202**2**
+* `aaaaaa` is the unique 6-digit number which is imprinted under the serial
+  number QR code
+
+#### System UUID
+
+SMBIOS specification defines a field for unique system identification with a
+special number called UUID (Universally Unique IDentifier). UUID is specified
+by [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122). MSI firmware
+provides the system UUID in the SMBIOS system information structure.
+
+The problem with UUID is that it cannot be recovered if the backup binary or
+SMBIOS logs are lost. You can backup the SMBIOS information with our
+[fwdump-docker](https://github.com/3mdeb/fwdump-docker). The `dmidecode.log`
```

Comment:
Maybe we will be able to point to DTS USB image, but we would need to test it first on the MSI.

Reply:
We have to I really hate technical debt we have with fwdump-docer. I asked couple times to archive that repo and switch to modern method that should be Dasharo state of the art DTS.

Reply:
Waiting for the link then...

Reply:
@miczyg1 See the #dasharo channel I asked for testing last weekend.

There were no volunteers, and we scheduled it for validation team

Reply:
Good, they should be the first ones to test it.

Reply:
Let's make sure everyone knows, this is blocking for this PR.

Reply:
Replaced with DTS link.